docs/modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -21,7 +25,7 @@ Ensure the WebDAV API is working::
   If all desktop clients fail to connect to the ownCloud Server, but access using the Web interface functions properly, the problem is often a misconfiguration of the WebDAV API.
   The ownCloud Client uses the built-in WebDAV access of the server content.
   Verify that you can log on to ownCloud's WebDAV server.
-  To verify connectivity with the ownCloud WebDAV server, open a browser window and enter the address to the ownCloud WebDAV server. For example, if your ownCloud instance is installed at `http://yourserver.com/owncloud`, your WebDAV server address isi `http://yourserver.com/owncloud/remote.php/webdav`. If you are prompted for your username and password but, after providing the correct credentials, authentication fails, please ensure that your authentication backend is configured properly.
+  To verify connectivity with the ownCloud WebDAV server, open a browser window and enter the address to the ownCloud WebDAV server. For example, if your ownCloud instance is installed at `\http://yourserver.com/owncloud`, your WebDAV server address isi `\http://yourserver.com/owncloud/remote.php/webdav`. If you are prompted for your username and password but, after providing the correct credentials, authentication fails, please ensure that your authentication backend is configured properly.
```

Comment:
```suggestion
  To verify connectivity with the ownCloud WebDAV server, open a browser window and enter the address to the ownCloud WebDAV server. For example, if your ownCloud instance is installed at `\http://yourserver.com/owncloud`, your WebDAV server address is `\http://yourserver.com/owncloud/remote.php/webdav`. If you are prompted for your username and password but, after providing the correct credentials, authentication fails, please ensure that your authentication backend is configured properly.
```

Reply:
I fixed it. just to let you know, this type was already existant before my PR...